src/components/Button/Button.css

```
@@ -0,0 +1,101 @@
+.App button{
```

Не используй класс из другого файла для стилизации отдельного компонента. Всегда для целей стилизации создавай класс под конкретный компонент (.button)
```suggestion
.button{
```